src/components/NavBar.css

```diff
@@ -0,0 +1,8 @@
+.NavBar{
+  background-color: #BB0032;
+  color: white;
+}
+
+a {
+  color: white !important;
+}
```

Comment:
this will target `a`s sitewide, is that intended?

Reply:
Removed the whole thing for now, as the links are not in, and will keep what you wrote in mind when I put the links in.